search/src/App.js

```diff
@@ -15,115 +15,138 @@ class App extends Component {
 	render() {
 		return (
 			<ReactiveBase
-			app="weedid"
-			url="http://localhost:9200/"
-			theme={{
-				typography: {
-				  fontFamily: 'Raleway, Helvetica, sans-serif',
-				},
-				colors: {
-				  primaryColor: '#0A0A0A',
-				  titleColor: '#E64626',
-				},
-				component: {
-				  padding: 10
-				}
-			  }}
+				app="weedid"
+				url="http://localhost:9200/"
+				theme={{
+					typography: {
+						fontFamily: 'Raleway, Helvetica, sans-serif',
+					},
+					colors: {
+						primaryColor: '#0A0A0A',
+						titleColor: '#E64626',
+					},
+					component: {
+						padding: 10
+					}
+				}}
 			>
-			<div style={{ display: "flex", flexDirection: "row" }}>
-			<div style={{ display: "flex", flexDirection: "column", width: "40%" }}>
-			    <CategorySearch
+				<div style={{ position: "fixed", width: "20rem", overflow: "scroll", height: "100%" }}>
+					{/* <DataSearch
```

Comment:
What's the benefit of keeping this commented section here? Can we remove it?

Reply:
No benefit; happy to cut. If we decide we want a similar generic "Search" bar, we can make one in the future that actually is useful.